tests/unit/plugins/kubernetes/kubernetes_pod_executor_test.py

```diff
@@ -21,12 +25,18 @@ def k8s_executor():
         autospec=True
     ), mock.patch.dict(os.environ, {"KUBECONFIG": "/this/doesnt/exist.conf"}):
         executor = KubernetesPodExecutor(namespace="task_processing_tests")
+        executor.api.create_namespaced_pod = mock.Mock()
```

Comment:
we probably won't need an extra mock here once we've moved to using kube_client
```suggestion
```

Reply:
switched to using kube_client